config/crd/apiextensions.k8s.io/v1/base/backup.appuio.ch_effectiveschedules.yaml

```diff
@@ -18,11 +18,11 @@ spec:
   versions:
   - additionalPrinterColumns:
     - description: Schedule Namespace
-      jsonPath: .spec.effectiveSchedules[0].namespace
+      jsonPath: .spec.scheduleRefs[0].namespace
```

Comment:
Can you put the generated things in their own commit?

Reply:
sure thing